src/js/base/settings.js

```diff
@@ -77,7 +77,7 @@ $.summernote = $.extend($.summernote, {
     toolbar: [
       ['style', ['style']],
       ['font', ['bold', 'underline', 'clear']],
-      ['fontname', ['fontname']],
+      ['fontname', ['fontname', 'fontsize']],
```

Comment:
This affects all Summernote instances which are working without custom toolbar option and it might cause other unintended side effects.